Bottom: f57db856b4433a530ba4c05ae7e36f5f54286e16
Top:    8b54c06e4f4dce1fd6e442ffdd7265db4cbea500
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-07-17 17:13:01 +0800

Refresh of vhost_net-introduce-tx

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index e6b74ba..657425b 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -494,8 +494,6 @@ static int get_tx_bufs(struct vhost_net *net,
 	int ret;
 
 	ret = vhost_net_tx_get_vq_desc(net, vq, out, in, busyloop_intr);
-	if (ret)
-		goto err;
 
 	if (*in) {
 		vq_err(vq, "Unexpected descriptor format for TX: out %d, int %d\n", *out, *in);
